zebra-network/src/peer_set/set.rs

```diff
@@ -472,3 +478,9 @@ where
         }
     }
 }
+
+// Get the type of a variable as a string
+use std::any::type_name;
+fn type_of<T>(_: T) -> &'static str {
```

Comment:
We will probably want to put this somewhere else if we end up keeping it.

Reply:
```suggestion
// replace with type_name_of_val when it stabilises
// https://github.com/rust-lang/rust/issues/66359
fn type_of<T>(_: T) -> &'static str {
```

Reply:
@yaahc might have an idea where we want to move this, but it's ok for now.

Reply:
Actually I think we can delete it and just check `Self`, see: https://github.com/ZcashFoundation/zebra/pull/1718/files#r573338707